noir/noir-repo/aztec_macros/src/transforms/functions.rs

```diff
@@ -588,8 +591,8 @@ fn abstract_storage(typ: &str, unconstrained: bool) -> Statement {
     assignment(
         "storage", // Assigned to
         call(
-            variable_path(chained_path!("Storage", "init")), // Path
-            vec![init_context_call],                         // args
+            variable_path(chained_path!(storage_struct_name.as_str(), "init")), // Path
+            vec![init_context_call],                                            // args
```

Comment:
Is this an unrelated bugfix in case the struct is not called `Storage`?

Reply:
Yes, I uses this PR for a little bit of cleanup